prog/mutation.go

```diff
@@ -337,6 +340,31 @@ func (t *BufferType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []
 	return
 }
 
+func mutateBufferSize(r *randGen, arg *DataArg, minLen, maxLen uint64) {
+	oldSize := arg.Size()
+	for oldSize == arg.Size() {
+		n := uint64(r.Intn(16)) + 1
+		if r.bin() {
+			// increase the size
+			if arg.Size() >= maxLen {
+				continue
+			}
+			if arg.Size()+n > maxLen {
+				n = maxLen
+			}
+		} else {
+			// decrease the size
+			if arg.Size() == 0 {
+				continue
+			}
+			if arg.Size()-n < minLen {
```

Comment:
`arg.Size()-n` can underflow since the type is unsigned, then it will become very large value.